MANIFESTO.md

```diff
@@ -32,6 +32,16 @@ This keeps the documents open for anyone to collaborate using state-of-the-art b
 Documents shall be pushed into a version control system like Git, *if and only if* it can be easily diff-ed by others, for the changes done.
 
 
+# Make docs functional
```

Comment:
Thanks @kopp for this contribution. This is a valid point.

I will hold this PR for a while as I plan to refactor the current Manifesto based on the feedback I received so far. The current Manifesto has more a guideline character in many places and not really a Manifesto.

A Manifesto shall talk about the values, principles and practices of docs-as-code. I will get this split done and then merge your changes to the practices part.